### PR TITLE
basic weather script

### PR DESCRIPTION
**Pull request by Baisang**
could also just use subprocess.call with curl instead of faking the user agent, not sure what is best approach

### CODE REVIEW

**Comment by chriskuehl** [comment]
why is this under desktop?

**Comment by chriskuehl** [comment]
otherwise lgtm

**Comment by Baisang** [comment]
no particular reason, didn't know where would be a better place to put

File: desktop/weather [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    # otherwise we get html in our response
+    headers = {'User-Agent': 'curl'}
+    res = requests.get(WTTR_SERVER + location, headers=headers)
+    if res.status_code == 200 and res.text != WTTR_ERROR:
+        print(res.text)
+    else:
+        print('location not found')
+
+def main(args):
+    parser = optparse.OptionParser()
```
Comment (chriskuehl) [inline desktop/weather]: argparse > optparse

**Comment by chriskuehl** [comment]
okay, i guess i don't have a better idea besides making a "lol" directory or something

**Comment by jvperrin** [comment]
Maybe put it in the `net` directory? This doesn't seem like it's desktop-specific, whereas `pdf-open` (the only current desktop script) is definitely desktop-specific.

**Comment by Baisang** [comment]
sounds good to me

**Comment by matthew-mcallister** [comment]
I am disappointed this does not work in tmux.

**Comment by matthew-mcallister** [comment]
Also you can decide weather you want to squash those commits.

**Comment by chriskuehl** [comment]
please don't start a pun thread
i can't take any more

**Comment by chriskuehl** [comment]
i'm deleting any future comments containing puns

**Comment by douglaslwong** [comment]
wtf is this

**Comment by gnowxilef** [comment]
why does it not work in tmux?

**Comment by chriskuehl** [comment]
wfm, provide details of failure?

**Comment by matthew-mcallister** [comment]
It doesn't work for me in tmux either:

screenshot - 12022016 - 11 13 05 pm

**Pull request merged**